File: back-end/Domain/BacklogItemServices/Commands/DTOs/BugAddUpdRequest.cs
using Raven.Yabt.Database.Common.BacklogItem;

namespace Raven.Yabt.Domain.BacklogItemServices.Commands.DTOs;

public record BugAddUpdRequest : BacklogItemAddUpdRequestBase
{
	public BugSeverity? Severity { get; set; }
	public BugPriority? Priority { get; set; }

	public string? StepsToReproduce { get; set; }
	public string? AcceptanceCriteria { get; set; }
}

File: front-end/src/app/core/api-models/backlog-item/item/BugAddUpdRequest.ts
import { BugPriority, BugSeverity } from '@core/api-models/common/backlog-item';
import { BacklogItemAddUpdRequestBase } from './BacklogItemAddUpdRequestBase';

export interface BugAddUpdRequest extends BacklogItemAddUpdRequestBase {
	severity: keyof typeof BugSeverity | undefined;
	priority: keyof typeof BugPriority | undefined;
	stepsToReproduce: string | undefined;
	acceptanceCriteria: string | undefined;
}
